lib/penelope/nlp/intent_classifier.ex

```diff
@@ -10,10 +10,11 @@ defmodule Penelope.NLP.IntentClassifier do
   utterances into a list of tokens. This pipeline is executed first, and
   its results are run through the classifier/recognizer pipelines.
 
-  Classification results are returned as a tuple of <intent, parameters>,
-  where intent is the name of the classified intent, and parameters is
-  a name->value map. Intent names, parameter names and parameter values are
-  all strings.
+  Classification results are returned as a tuple of <intents, parameters>,
```

Comment:
Would it make sense to markdownify this for hex? `{intents, parameters}`?

Reply:
Maybe, I just used the math notation for tuples.